doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -157,32 +155,37 @@ Boundary Conditions
 ~~~~~~~~~~~~~~~~~~~
 .. code-block:: text
 
-    subsection boundary conditions
-      set number = 5
-      subsection bc 0
-        set id   = 0
-        set type = noslip
-      end
-      subsection bc 1
-        set id   = 1
-        set type = noslip
-      end
-    
-      subsection bc 2
-        set id   = 2
-        set type = noslip
-      end
-      subsection bc 3
-        set id   = 4
-        set type = noslip
-      end
-      subsection bc 4
-        set id   = 5
-        set type = noslip
-      end
+  subsection boundary conditions
+    set number = 6
+    subsection bc 0
+      set id   = 0
+      set type = noslip
+    end
+    subsection bc 1
+      set id   = 1
+      set type = noslip
     end
 
-Here we define the 5 ``no slip`` boundary for all the box walls and let the boundary with ``id=3`` free to represent the top of the box. We refer the reader to the :doc:`../../../parameters/cfd/boundary_conditions_cfd` section on how those boundaries are defined. 
+    subsection bc 2
+      set id   = 2
+      set type = noslip
+    end
+    subsection bc 3
+      set id   = 4
+      set type = noslip
+    end
+    subsection bc 4
+      set id   = 5
+      set type = noslip
+    end
+    subsection bc 5
+      set id   = 3
+      set type = outlet
+      set beta = 0
+    end
+  end
```

Comment:
I would use this opportunity to renumber the boundary ids so that they match the mesh ids. It would be simpler to read and make more sense.